### PR TITLE
APPS-963 Use array to store request's bin list

### PR DESCRIPTION
**Pull request by korotkov-aerospike**

### CODE REVIEW

File: pkg/model/config.go [hunk 1]
```diff
@@ -121,8 +121,8 @@ type BackupPolicy struct {
 	Namespace          *string     `yaml:"namespace,omitempty" json:"namespace,omitempty"`
 	Parallel           *int32      `yaml:"parallel,omitempty" json:"parallel,omitempty"`
 	SetList            *[]string   `yaml:"set_list,omitempty" json:"set_list,omitempty"`
+	BinList            *[]string   `yaml:"bin_list,omitempty" json:"bin_list,omitempty"`
 	NodeList           *string     `yaml:"node_list,omitempty" json:"node_list,omitempty"`
```
Comment (reugn) [inline pkg/model/config.go:125]: Make it an array as well, use the `[]string` type for all of the array fields

File: pkg/model/config.go
Comment (reugn) [inline pkg/model/config.go:1]: * `PartitionList` can also be an array. Check whether partition ranges are supported.

Comment (korotkov-aerospike) [reply pkg/model/config.go:1]: No,  unfortunately

`List of partitions to back up. Partition filters can be ranges, individual partitions, or records after a specific digest within a single partition`

File: pkg/shared/backup.go [hunk 2]
```diff
@@ -22,6 +22,8 @@ import (
 	"time"
 	"unsafe"
 
+	"github.com/aws/smithy-go/ptr"
```
Comment (reugn) [inline pkg/shared/backup.go:25]: Move this import to the bottom

File: pkg/model/config.go [hunk 3]
```diff
@@ -120,9 +120,9 @@ type BackupPolicy struct {
 	Storage            *string     `yaml:"storage,omitempty" json:"storage,omitempty"`
 	Namespace          *string     `yaml:"namespace,omitempty" json:"namespace,omitempty"`
 	Parallel           *int32      `yaml:"parallel,omitempty" json:"parallel,omitempty"`
-	SetList            *[]string   `yaml:"set_list,omitempty" json:"set_list,omitempty"`
-	NodeList           *string     `yaml:"node_list,omitempty" json:"node_list,omitempty"`
-	BinList            *string     `yaml:"bin_list,omitempty" json:"bin_list,omitempty"`
+	SetList            []string    `yaml:"set_list,omitempty" json:"set_list,omitempty"`
+	BinList            []string    `yaml:"bin_list,omitempty" json:"bin_list,omitempty"`
+	NodeList           []string    `yaml:"node_list,omitempty" json:"node_list,omitempty"`
```
Comment (reugn) [inline pkg/model/config.go:125]: Use this field in the backup configuration.

Comment (korotkov-aerospike) [reply pkg/model/config.go]: I think, it would be better to have structure for Node

**Review by reugn — APPROVED** [review]

**Pull request merged**